worker/agentconfigupdater/manifold.go

```diff
@@ -103,8 +117,8 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 				}
 				config.SetStateServingInfo(info)
 				if mongoProfileChanged {
-					logger.Debugf("setting agent config mongo memory profile: %s", mongoProfile)
-					config.SetMongoMemoryProfile(mongoProfile)
+					logger.Debugf("setting agent config mongo memory profile: %s", configMongoMemoryProfile)
```

Comment:
maybe it would be nice to see:

"setting agent config mongo memory profile: %s => %s", agentMongoMemoryProfile, configMongoMemoryProfile)

you have the information.
Thoughts?

Reply:
Seems reasonable to me. Updated.